.github/workflows/retest.yaml

```diff
@@ -9,8 +9,7 @@ permissions:
 
 jobs:
   retest:
-    if: |
-      ${{
+    if: ${{
```

Comment:
this was the error detected by it

```
.github/workflows/retest.yaml:12:9: if: condition "${{\n   github.event.issue.pull_request\n   && github.repository == 'envoyproxy/ai-gateway'\n   && github.actor != 'repokitteh-read-only[bot]'\n   && github.actor != 'dependabot[bot]'\n}}\n" is always evaluated to true because extra characters are around ${{ }} [if-cond]
   |
12 |     if: |
   |         ^
exit status 1

```